packages/styles/scss/_typography.scss

```diff
@@ -10,35 +10,32 @@
 html,
 body {
   color: $color-font-base;
-  font: #{$type-base-size}/#{$type-base-line-height} $fonts-copy;
+  font-family: var(--ilo-fonts-copy);
+  font-size: #{$type-base-size};
+  line-height: #{$type-base-line-height};
   letter-spacing: $type-base-letter-spacing;
   -webkit-font-smoothing: antialiased;
 }
 
-/// Font family fallbacks for: Overpass, Noto Sans
-/// @type Map
-/// @access public
-/// @group @carbon/type
-$font-families: (
-  "display": unquote("#{$fonts-display}, sans-serif"),
-  "copy": unquote("#{$fonts-copy}, sans-serif"),
-) !default;
+h1,
+h2,
+h3,
+h4,
+h5,
+h5 {
+  font-family: var(--ilo-fonts-display);
+}
 
-/// Get the font-family for a font
-/// @param {String} $name
-/// @return {String}
-/// @access public
-/// @group @carbon/type
-@function font-family($name) {
-  @return map-get($font-families, $name);
+// Set Chinese system fonts
+html[lang="zh"] {
+  --ilo-fonts-display: var(--ilo-fonts-display-zh);
+  --ilo-fonts-copy: var(--ilo-fonts-copy-zh);
 }
 
-/// Include the `font-family` definition for the given name in your selector
-/// @param {String} $name
-/// @access public
-/// @group @carbon/type
-@mixin font-family($name) {
-  font-family: font-family($name);
+// Set Japanese system fonts
+html[lang="jp"] {
```

Comment:
Was the design system tested with all the languages that it should support? I have a feeling that there can be similar issues in other languages

Reply:
@bashlk it was tested with Latin languages, Cyrillic languages and Arabic. Chinese and Japanese weren't tested adequately. What makes them different is that you don't ship Chinese and Japanese fonts, they have tens of thousands of glyphs so webfonts aren't really a thing for those languages. You're stuck using system fonts in those cases, unlike languages like Arabic where you have a discreet character set that you can support with webfonts.